miscellaneous/redirect.nowhere

```diff
@@ -0,0 +1,13 @@
+# bind9 zone file template to redirect spam domains to nowhere aka
+#   block them
+
+$TTL    600
+@                       1D IN SOA       example.com hostmaster.example (
+                                        42              ; serial
+                                        3H              ; refresh
+                                        15M             ; retry
+                                        1W              ; expiry
+                                        1D )            ; minimum
+
+# @	IN	NS	<YourNameServerIP>
+*	IN	A	0.0.0.0
```

Comment:
Change this to 

```
*	IN	CNAME	.
```

Next to this you should then keep a wildcard list to take full advantage of the [RPZ](https://mypdns.org/mypdns/support/-/wikis/RPZ)

Record types https://mypdns.org/mypdns/support/-/wikis/RPZ-record-types

Zone example can be found here: 

drill @ns1.mypdns.org AXFR adult.mypdns.cloud

Bind setup https://mypdns.org/rpz/dns-rpz-integration/-/tree/master/Bind_9

Why all this, well RPZ simply just rocks when you take the full advantage of all it capabilities.

Example of true power, https://mypdns.org/my-privacy-dns/matrix/-/issues?scope=all&state=closed&label_name[]=DNS%20Server&label_name[]=Pirated These DNS server are blacklisted and in combination these are blocking access to over 2 million pirated domains, and if you are into NSFW blacklisting you can always add these https://mypdns.org/my-privacy-dns/porn-records/-/issues?scope=all&state=all&label_name[]=TLD%20(Top%20Level%20Domain)

Have fun experiencing "new" stuff :wave: